templates/web/base/report/_report_meta_info.html

```diff
@@ -1,2 +1,6 @@
 [% problem.meta_line(c) | html %]
+[% IF c.user_exists AND c.user.id == problem.user_id AND !problem.anonymous %]
+  <!-- TODO: Should this link to a page with server-side form, for users without JavaScript? -->
```

Comment:
For now, set it up as an `/about/` page (ie. http://fixmystreet.org/customising/static-pages/ ) and we can hook/move it from there.